Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -115,6 +115,16 @@ open class Player: AVPlayerViewController {
         }
     }
 
+    public var chromelessMode = false {
```

Comment:
Chromeless mode control is now split in two separate properties: `isChromeless` and `chromelessMode`. They reflect the same state but are not synchronized. This leads to inconsistencies in the code (e.g.: the check for `isChromeless` and not for `chromelessMode` in `willEnterForeground`).

I recommend keeping this state in a single place and making sure the `kChromeless` option received on the Player scope is only the initial value for this state.

Reply:
We merge `isChromeless` with `chromelessMode` and now we are keeping this state in a single place.